data/hai/hai ships.txt

```diff
@@ -177,6 +177,62 @@ ship "Centipede" "Centipede (Refugees)"
 
 
 
+ship "Cicada"
+	sprite "ship/hai cicada"
+	thumbnail "thumbnail/hai cicada"
+	attributes
+		category "Transport"
+		"cost" 5850000
+		"shields" 4900
+		"hull" 2800
+		"required crew" 7
+		"bunks" 51
+		"mass" 240
+		"drag" 4.7
+		"heat dissipation" .75
+		"fuel capacity" 600
+		"cargo space" 143
+		"outfit space" 323
+		"weapon capacity" 89
+		"engine capacity" 58
+		weapon
+			"blast radius" 80
+			"shield damage" 960
+			"hull damage" 420
+			"hit force" 1280
+	outfits
+		"Hai Tracker Pod"
+		"Hai Tracker" 84
+		"Tracker Storage Pod"
+		"Bullfrog Anti-Missile" 2
+		"Chameleon Anti-Missile" 2
+		"Geode Reactor"
+		"Hai Gorge Batteries"
+		"Hai Diamond Regenerator"
+		"Hai Williwaw Cooling"
+		"Quantum Keystone"
+		`"Benga" Atomic Thruster`
+		`"Benga" Atomic Steering`
+		"Hyperdrive"
+	engine -6 75
+	engine 6 75
+	gun 0 -76 "Hai Tracker Pod"
+	turret 34.5 -16.5 "Chameleon Anti-Missile"
+	turret -34.5 -16.5 "Chameleon Anti-Missile"
+	turret -20 42 "Bullfrog Anti-Missile"
+	turret 20 42 "Bullfrog Anti-Missile"
+	leak "leak" 50 30
+	leak "flame" 40 50
+	leak "big leak" 20 40
+	explode "small explosion" 11
+	explode "medium explosion" 24
+	explode "large explosion" 16
+	explode "huge explosion" 4
+	"final explode" "final explosion medium"
+	description `Practically no new Cicadas have been built in recent memory. A design nearly as old as the Aphid; it is so reliable that the second-hand market is filled with Cicadas that have outlived not one, but in many cases multiple whole lineages of Hai that have owned one and passed it down through the generations. This feature is likely the main factor which has kept the popularity of the Scarab in check as well as defining the Yeer e Ki shipyard's motto: "Service for Life".`
```

Comment:
```suggestion
	description `A design nearly as old as the Aphid, practically no new Cicadas have been built in recent memory; it is so reliable that the second-hand market is saturated Cicadas that have outlived not just one pilot, but in many cases multiple whole lineages of Hai that have owned one and passed it down through the generations. This feature is likely the main factor which has kept the popularity of the Scarab in check as well as defining the Yeer e Ki shipyard's motto: "Service for Life".`
```
	
These are strictly style things, but it does incorporates the above recommendation as well.

Reply:
I was trying not to rip the exact same phrasing as the Waterbug uses. If people prefer this though then it's fine.